### PR TITLE
Safer sql awscostentrylineitem daily

### PR DESCRIPTION
**Pull request by nbon12**
Story: https://github.com/project-koku/koku/issues/1079
Iteratively improving the safety of SQL statements, this PR introduces the usage of SQL parameters separately passed into the database, rather than SQL and variables in the same string. Using [JinjaSQL](https://github.com/hashedin/jinjasql) to generate the variables list separately from the SQL

### CODE REVIEW

File: koku/masu/database/aws_report_db_accessor.py [hunk 1]
```diff
@@ -197,23 +197,27 @@ def populate_line_item_daily_table(self, start_date, end_date, bill_ids):
 
         """
         table_name = AWS_CUR_TABLE_MAP['line_item_daily']
+        from jinjasql import JinjaSql
+        j = JinjaSql()
```
Comment (nbon12) [inline koku/masu/database/aws_report_db_accessor.py]: I'm planning to move this out of the function into the heading of the file

File: koku/masu/database/report_db_accessor_base.py [hunk 2]
```diff
@@ -424,7 +424,10 @@ def _commit_and_vacuum(self, table, sql, start=None, end=None):
             transaction.savepoint_commit(KokuDBAccess._savepoints.pop())
         with connection.cursor() as cursor:
             cursor.db.set_schema(self.schema)
-            cursor.execute(sql)
+            if bind_params is not None:
+                cursor.execute(sql, bind_params)
+            else:
+                cursor.execute(sql)
```
Comment (blentz) [inline koku/masu/database/report_db_accessor_base.py]: I don't think a conditional is needed here. This should be functionally equivalent and using a keyword arg means not needing to care about whether `bind_params` is None or not:

```suggestion
                cursor.execute(sql, bind_params=bind_params)
```

File: koku/masu/database/aws_report_db_accessor.py [hunk 3]
```diff
@@ -197,23 +198,26 @@ def populate_line_item_daily_table(self, start_date, end_date, bill_ids):
 
         """
         table_name = AWS_CUR_TABLE_MAP['line_item_daily']
+
+        jinjaSql = JinjaSql()
```
Comment (adberglund) [inline koku/masu/database/aws_report_db_accessor.py]: This is a nitpick, but consistent with our code. Use lowercase where it makes sense and separate with underscores. 
```suggestion
        jinja_sql = JinjaSql()
```

File: koku/masu/database/report_db_accessor_base.py (hunk 2 above)
Comment (nbon12) [reply koku/masu/database/report_db_accessor_base.py]: change added

**Comment by nbon12** [comment]
FYI, table names are not made SQL-safe in this PR, see https://github.com/hashedin/jinjasql#sql-safe-strings

**Pull request merged**